lib/stream_data.ex

```diff
@@ -1156,20 +1156,22 @@ defmodule StreamData do
 
   ## Examples
 
-      Enum.take(StreamData.map_of(StreamData.integer(), StreamData.boolean()), 3)
+      Enum.take(StreamData.map_of(StreamData.integer(), StreamData.boolean()), max_tries: 3)
```

Comment:
The `3` argument was to `Enum.take/2`, so it was correct

Reply:
Good catch.